### DISCUSSION TITLE
How do I get the value of the "X-RateLimit-Remaining" field in rate-limiting by calling an api?

### DISCUSSION THREAD

**Discussion by ichuan258** [Kong/kong#8086] [Help]
I need to get the value of "X-RateLimit-Remaining", can I get this by calling the GET method? I know this requires code modification, so I need a direction
```
HTTP/1.1 200 OK
Content-Type: application/json; charset=utf-8
Content-Length: 354
Connection: keep-alive
X-RateLimit-Remaining-Minute: 9
X-RateLimit-Limit-Minute: 10
RateLimit-Limit: 10
RateLimit-Remaining: 9
RateLimit-Reset: 41
Cache-Control: no-cache
Date: Wed, 17 Nov 2021 01:58:25 GMT
X-Kong-Upstream-Latency: 15
X-Kong-Proxy-Latency: 6
Via: kong/2.6.0
```
I wish to call ``` GET  plugins/{uuid}``` to Get the value of this field ```X-RateLimit-Remaining-Minute```

**Comment by pintsized** [comment]
There's not a central admin API place for this value, since the value is somewhat dynamic, based on what you are limiting by. For example, with `limit_by` set to `IP`, which address should it use to calculate the reply? In other words, the value you are looking for depends on the nature of the request itself.

The only way to know this type of information is to invoke the plugin as a normal request, and inspect the metadata in the headers.
